model/registry/scm.yaml

```diff
@@ -0,0 +1,37 @@
+groups:
+  - id: registry.scm.repository
+    prefix: scm
+    type: attribute_group
+    brief: >
+     This group defines the attributes for source control management (SCM) systems.
+    attributes:
+      - id: name
```

Comment:
do we need an attribute that would describe a repo provider? GitHub, GitLab, git, mercurial, etc (suggesting `scm.system` to match similar attributes)

Reply:
@christophe-kamphaus-jemmic  asked this same thing and mentioned the potential new [entity model](https://github.com/open-telemetry/oteps/pull/256) coming. Do you have thoughts here? In our gitprovider receiver we emit a resource attribute called `git.vendor` to denote if it's coming from `GitHub/GitLab/etc`

Reply:
closing in favor of this issue after discussing in the previous sem cov wg #1193